### PR TITLE
Support jdbc tables in convention tests

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

**Comment by losipiuk** [comment]
cc: @brian-rickman

File: tempto-core/src/main/java/com/teradata/tempto/internal/convention/SqlDescriptor.java [hunk 1]
```diff
@@ -35,6 +36,11 @@
     protected final SectionParsingResult sqlSectionParsingResult;
     private final Map<String, String> baseProperties;
 
+    protected SqlDescriptor(SectionParsingResult sqlSectionParsingResult)
+    {
+        this(sqlSectionParsingResult, Collections.emptyMap());
```
Comment (kokosing) [inline tempto-core/src/main/java/com/teradata/tempto/internal/convention/SqlDescriptor.java]: static import

File: tempto-examples/src/main/resources/sql-tests/datasets/sample_jdbc_table.data-revision [hunk 2]
```diff
@@ -0,0 +1 @@
+v.1.0
```
Comment (kokosing) [inline tempto-examples/src/main/resources/sql-tests/datasets/sample_jdbc_table.data-revision]: new lines

**Comment by kokosing** [comment]
looks good

**Pull request merged**